app/views/parks/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Hello from parks show!</h1>
+
+<div id="app" data-id= <%= @id %> >
```

Comment:
![I see what you did there.](https://media.giphy.com/media/CcUk4a6fkgUfu/giphy.gif)